src/Catty/ViewController/Continue&New/MaintainObject/MaintainScript/BrickCategoryOverviewController.swift

```diff
@@ -0,0 +1,130 @@
+/**
+ *  Copyright (C) 2010-2020 The Catrobat Team
+ *  (http://developer.catrobat.org/credits)
+ *
+ *  This program is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU Affero General Public License as
+ *  published by the Free Software Foundation, either version 3 of the
+ *  License, or (at your option) any later version.
+ *
+ *  An additional term exception under section 7 of the GNU Affero
+ *  General Public License, version 3, is available at
+ *  (http://developer.catrobat.org/license_additional_term)
+ *
+ *  This program is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ *  GNU Affero General Public License for more details.
+ *
+ *  You should have received a copy of the GNU Affero General Public License
+ *  along with this program.  If not, see http://www.gnu.org/licenses/.
+ */
+
+import UIKit
+
+@objc class BrickCategoryOverviewController: UIViewController, UICollectionViewDelegate, UICollectionViewDataSource {
+
+    var brickCategoryOverviewCollectionView: UICollectionView!
+    var categegoriesBricks = [BrickCategory]()
+    var scvc: ScriptCollectionViewController?
+
+    // MARK: Init
+    @objc override init(nibName nibNameOrNil: String?, bundle nibBundleOrNil: Bundle?) {
```

Comment:
I think this initializer is unused.

Reply:
It's needed because its a required initialiser and the data of the brick is loaded here.

Reply:
Yes but why do we need it here, while we are not really using it? Without this initializer you can make the `scvc` property non-optional which would make a lot of sense.